pom.xml

```diff
@@ -1,976 +1,646 @@
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-cg</artifactId>
+            <version>${kfs.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-kns</artifactId>
+            <version>${kfs.version}</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-core</artifactId>
+            <version>${kfs.version}</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-core</artifactId>
+            <version>${kfs.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-ec</artifactId>
+            <version>${kfs.version}</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-ec</artifactId>
+            <version>${kfs.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-ld</artifactId>
+            <version>${kfs.version}</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-ld</artifactId>
+            <version>${kfs.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-purap</artifactId>
+            <version>${kfs.version}</version>
+            <type>jar</type>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-purap</artifactId>
+            <version>${kfs.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.kuali.kfs</groupId>
+            <artifactId>kfs-web</artifactId>
+            <version>${kfs.version}</version>
+            <type>war</type>
+        </dependency>
+        <dependency>
             <groupId>org.kuali.kfs</groupId>
-            <artifactId>kfs-lib-test</artifactId>
+            <artifactId>kfs-web</artifactId>
```

Comment:
since there was a "Test" characteristic to the artifact ID before, shouldn't there be one now?